### DISCUSSION TITLE
Enable the ability to have multiple Carts per Session

### DISCUSSION THREAD

**Discussion by andris-sevcenko** [craftcms/commerce#2298] [Ideas]
> Created by: Steve Holland ([steve.holland@mac.com](steve.holland@mac.com)) on 2016/06/01 08:04:40 +0000
Votes at time of UserVoice import: 2

------
This would allow you to create more of an "Etsy" style marketplace site. 

You would be able to have Products/Shipping Methods/Discounts associated with a Seller. Then when Products are added to the Cart, the Cart is associated with a particular Seller. If Products are added form a different Seller a new Cart is added to the Collection. On Checkout you could enter one set of Payment details, but send payments to multiple sellers using something like Stripe Connect.

**Comment by cimocimocimo** [comment]
I have another use case that this would be good for. Subscription service that also has a store with regular purchases. Carts can be used for customizing the monthly/quarterly subscription items and for buying one-off purchases from the regular store. When the Subscription items are ready to ship we'd complete the Order and then the subscribers wouldn't be able customize their orders any longer.
